postman/Get Broadcast Open Clicks.postman_collection.json

```diff
@@ -0,0 +1,336 @@
+{
+	"info": {
+		"_postman_id": "886568e4-566c-42d1-833e-58b3de8c0833",
+		"name": "Get Broadcast Open Clicks",
```

Comment:
I think this collection should be called Get Broadcast Links

Reply:
Renamed